apps/settings/elements/root.html

```diff
@@ -144,6 +144,11 @@ <h2 data-l10n-id="accounts">Accounts</h2>
             <small class="findmydevice-desc menu-item-desc"></small>
           </a>
         </li>
+        <li>
+          <a id="menuItem-achievements" class="menu-item" data-icon="achievements" href="#achievements">
```

Comment:
The icon seems not ready yet. We should wait for this icon ready before merging as this is a first level menu item.